istioctl/pkg/describe/describe.go

```diff
@@ -906,13 +906,18 @@ func printIngressInfo(
 	pod := pods.Items[0]
 
 	// Currently no support for non-standard gateways selecting non ingressgateway pods
-	ingressSvcs, err := kubeClient.CoreV1().Services(istioNamespace).List(context.TODO(), metav1.ListOptions{
-		LabelSelector: "istio=ingressgateway",
-	})
+	ingressSvcs, err := kubeClient.CoreV1().Services(istioNamespace).List(context.TODO(), metav1.ListOptions{})
 	if err != nil {
 		return multierror.Prefix(err, "Could not find ingress gateway service")
 	}
-	if len(ingressSvcs.Items) == 0 {
+	filteredIngressSvcs := []corev1.Service{}
+	for _, svc := range ingressSvcs.Items {
+		svc := svc
```

Comment:
What's the purpose of this re-assigment?

Reply:
Thanks, forgot deleting this, it's not necessary here.